src/main/java/picard/analysis/directed/CollectTargetedMetrics.java

```diff
@@ -60,42 +60,47 @@ protected abstract COLLECTOR makeCollector(final Set<MetricAccumulationLevel> ac
                                                final int nearProbeDistance);
 
 
-    @Option(shortName = "TI", doc = "An interval list file that contains the locations of the targets.", minElements=1)
+    @Argument(shortName = "TI", doc = "An interval list file that contains the locations of the targets.", minElements=1)
     public List<File> TARGET_INTERVALS;
 
-    @Option(shortName = StandardOptionDefinitions.INPUT_SHORT_NAME, doc = "An aligned SAM or BAM file.")
+    @Argument(shortName = StandardOptionDefinitions.INPUT_SHORT_NAME, doc = "An aligned SAM or BAM file.")
     public File INPUT;
 
-    @Option(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "The output file to write the metrics to.")
+    @Argument(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "The output file to write the metrics to.")
     public File OUTPUT;
 
-    @Option(shortName = "LEVEL", doc = "The level(s) at which to accumulate metrics.")
+    @Argument(shortName = "LEVEL", doc = "The level(s) at which to accumulate metrics.")
     public Set<MetricAccumulationLevel> METRIC_ACCUMULATION_LEVEL = CollectionUtil.makeSet(MetricAccumulationLevel.ALL_READS);
 
-    @Option(optional = true, doc = "An optional file to output per target coverage information to.")
+    @Argument(optional = true, doc = "An optional file to output per target coverage information to.")
     public File PER_TARGET_COVERAGE;
 
-    @Option(optional = true, doc = "An optional file to output per base coverage information to. The per-base file contains " +
+    @Argument(optional = true, doc = "An optional file to output per base coverage information to. The per-base file contains " +
             "one line per target base and can grow very large. It is not recommended for use with large target sets.")
     public File PER_BASE_COVERAGE;
 
-    @Option(optional = true, doc= "The maximum distance between a read and the nearest probe/bait/amplicon for the read to be " +
+    @Argument(optional = true, doc= "The maximum distance between a read and the nearest probe/bait/amplicon for the read to be " +
             "considered 'near probe' and included in percent selected.")
     public int NEAR_DISTANCE = TargetedPcrMetricsCollector.NEAR_PROBE_DISTANCE_DEFAULT;
 
-    @Option(shortName = MINIMUM_MAPPING_QUALITY_SHORT_NAME, doc = "Minimum mapping quality for a read to contribute coverage.", overridable = true)
+    //TODO: MINIMUM_MAPPING_QUALITY and MINIMUM_BASE_QUALITY are both annotated as required,
```

Comment:
Barclay retains this, um, feature, and works the same way. It causes some [problems](https://github.com/broadinstitute/barclay/issues/23) though so I'd like to change it at some point.

Anyway, for now Ill make no change other than to remove my comment.